manim/scene/three_d_scene.py

```diff
@@ -42,7 +42,9 @@ def __init__(
         )
         super().__init__(camera_class=camera_class, **kwargs)
 
-    def set_camera_orientation(self, phi=None, theta=None, distance=None, gamma=None):
+    def set_camera_orientation(
+        self, phi=None, theta=None, distance=None, gamma=None, frame_center=None
+    ):
```

Comment:
**(suggestion)**: Since you've added a new a paremeter, it should have some docs explaining its purpose. Also, adding some typings (https://docs.manim.community/en/stable/contributing/typings.html)  + moving the types from the docstring to the definition would be appreciated.

Reply:
Done, but only for the functions I've changed. Still not certain whether it's done correctly.